bindings/matrix-sdk-crypto-js/Cargo.toml

```diff
@@ -31,9 +31,12 @@ matrix-sdk-common = { version = "0.5.0", path = "../../crates/matrix-sdk-common"
 matrix-sdk-crypto = { version = "0.5.0", path = "../../crates/matrix-sdk-crypto" }
 ruma = { git = "https://github.com/ruma/ruma", rev = "96155915f", features = ["client-api-c", "js", "rand", "unstable-msc2676", "unstable-msc2677"] }
 vodozemac = { git = "https://github.com/matrix-org/vodozemac/", rev = "2404f83f7d3a3779c1f518e4d949f7da9677c3dd", features = ["js"] }
-wasm-bindgen = "0.2.80"
-wasm-bindgen-futures = "0.4.30"
-js-sys = "0.3.49"
-serde_json = "1.0.79"
-http = "0.2.6"
+wasm-bindgen = "0.2"
+wasm-bindgen-futures = "0.4"
+js-sys = "0.3"
+console_error_panic_hook = "0.1"
+serde_json = "1.0"
+http = "0.2"
```

Comment:
why are you removing the minor and patch level versions of these? We generally follow: add version at mininum supported version, which more often than not means we just use the version at the time of writing. By removing the minor, the cargo resolver might resolve to a different, lower, version because of other dependencies pinning, which in turn might break us, as we required a feature added in the major ... IMHO it is better to break at explicit version resolving in this case rather than successfully resolve and then break on missing features (as the later, is super hard to understand and spot for the developer, too).